packages/botonic-cli/src/commands/deploy.ts

```diff
@@ -277,24 +278,24 @@ Uploading...
       text: 'Creating bundle...',
       spinner: 'bouncingBar',
     }).start()
-    // Why using rimraf instead of fs.rmdirSync?: https://stackoverflow.com/a/57834155/145289
-    if (fs.existsSync('tmp')) rimraf.sync('tmp')
-    fs.mkdirSync(join('tmp'))
-    copySync('dist', join('tmp', 'dist'))
+    if (pathExists('tmp')) remove('tmp')
```

Comment:
if for some reason the temporary directory is not removed, with createTempDir we are going to generate more than one temporary directories in the developer project. What about calling it `botonic-tmp` in order to minimize the probability of collisioning with an already existing `tmp`?

Reply:
Well, createTempDir could be in the top of the cli command with a finally to remove it. And leaving rubbish in tmp dir is not so bad because user will eventually remove it. I prefer rubbish there rather than in my bot project.
But in this PR let's **not do anything** please. Just create a linear with a copy of this discussion and let's think about it

Reply:
annotated 👍